shell/components/__tests__/ContainerResourceLimit.test.ts

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils';
+import ContainerResourceLimit from '@shell/components/ContainerResourceLimit.vue';
+
+describe('component: ContainerResourceLimit', () => {
+  it.skip.each([
```

Comment:
Do we intend to skip here?

Reply:
yes, it's not rendering and I don't know why, but this test has to be there